lib/systems/default.nix

```diff
@@ -66,6 +66,44 @@ rec {
          # uname -r
          release = null;
       };
+
+      qemuArch = if final.isArm then "arm"
+        else {
+          "armv5tel" = "arm";
+          "armv6l" = "arm";
+          "armv7l" = "arm";
+          "armv7a" = "arm";
+          "i686" = "i386";
+          "powerpc64le" = "ppc64";
+        }.${final.parsed.cpu.name} or final.parsed.cpu.name;
+
+      emulator = pkgs: let
```

Comment:
@ericson2314 How do you feel about putting this in lib/systems/default.nix? I want to avoid sticking it in "pkgs" as that is only really supposed to contain callPackage usages.

Reply:
Can you move the emulator stuff to its own file?